canonical/ubuntu.com
# Update /telco page

## Done

- Updates based on changed in [copydoc](https://docs.google.com/document/d/1a3otoXCbXtddJG2HCnYIIHJiDxvG-pzFPtLobVB4-vw/edit)
- Updated UX in call with Juan :)

## QA

- Check the changes outlined in the [copydoc](https://docs.google.com/document/d/1a3otoXCbXtddJG2HCnYIIHJiDxvG-pzFPtLobVB4-vw/edit) are reflected in [the demo](https://ubuntu-com-13787.demos.haus/telco)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-10363

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
